packages/gasket-plugin-elastic-apm/lib/index.js

```diff
@@ -46,41 +45,36 @@ module.exports = {
         );
 
         if (!apm.isStarted()) {
-          logger.notice('WARNING Elastic APM agent is not started. Use `--require elastic-apm-node/start`');
+          logger.notice('WARNING Elastic APM agent is not started. Use `--require ./setup.js`');
         }
 
-        apm.addFilter(filterSensitiveCookies(config));
+        if (config.elasticAPM && config.elasticAPM.sensitiveCookies) {
+          logger.notice('WARNING: elasticAPM.sensitiveCookies has been removed. Filter sensitive data in the setup.js script.');
```

Comment:
This probably warrants a `warning` level log
```suggestion
          logger.warning('elasticAPM.sensitiveCookies has been removed. Filter sensitive data in the setup.js script.');
```